### DISCUSSION TITLE
Allow file extensions on asset URL

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#4036] [Feature Requests]
When loading an image using the asset URL `https://example.co.uk/assets/12345678-1234-1234-1234-123456789012` allow the use of file extensions for compatibility when including images an example being `https://example.co.uk/assets/12345678-1234-1234-1234-123456789012.jpg` or `https://example.co.uk/assets/12345678-1234-1234-1234-123456789012.png`

**Comment by philleepflorence** [comment]
I concur, the option would help in multiple cases, as well as make [Assets](https://docs.directus.io/reference/api/assets/#accessing-an-original-file) more agnostic, without using **Direct File Access**.

**Comment by rijkvanzanten** [comment]
This will be implemented, and is tracked in https://github.com/directus/directus/issues/4067
